Tests/SentryTests/SentryViewPhotographerTests.swift

```diff
@@ -210,6 +237,44 @@ class SentryViewPhotographerTests: XCTestCase {
         assertColor(pixel1, .green)
     }
     
+    func testNotMaskingLabelInsideclippedViewHiddenByAnOpaqueExternalView() throws {
+        let topView = UIView(frame: CGRect(x: 25, y: 0, width: 25, height: 25))
+        topView.backgroundColor = .green
+        
+        let label1 = UILabel(frame: CGRect(x: 0, y: 0, width: 50, height: 25))
+        label1.text = "Test"
+        label1.textColor = .black
+        
+        let parentView = UIView(frame: CGRect(x: 0, y: 0, width: 50, height: 25))
+        parentView.addSubview(label1)
+        parentView.clipsToBounds = true
+        
+        let image = try XCTUnwrap(prepare(views: [parentView, topView]))
```

Comment:
`m`: It would be highly beneficial to store the image to a temp location for debugging purposes or when a test fails to understand better what's going on with something like:
```
if let data = image.pngData() {
    let url = FileManager.default.temporaryDirectory.appendingPathComponent("testImage.png")
    try? data.write(to: url)
    print("Image saved to: \(url.path)")
}
```